src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1622,6 +1662,44 @@ private void RemoveOrganelleAt(Hex location)
         EnqueueAction(action);
     }
 
+    [DeserializedCallbackAllowed]
+    private void DoOrganelleMoveAction(MicrobeEditorAction action)
+    {
+        var data = (MoveActionData)action.Data;
+        editedMicrobeOrganelles.Remove(data.Organelle);
```

Comment:
I think that when a move starts, the organelle should then be removed from the organelle otherwise I don't think this works correctly:
check that organelle move allows big organelle to be moved to be slightly overlapping the existing position (might need to remove from the species, and then the moved organelle *must* be saved)